doc/Jamfile.v2

```diff
@@ -0,0 +1,38 @@
+#   Copyright (c) 2002 Douglas Gregor <doug.gregor -at- gmail.com>
+
+#   Distributed under the Boost Software License, Version 1.0.
+#   (See accompanying file LICENSE_1_0.txt or copy at
+#   http://www.boost.org/LICENSE_1_0.txt)
+
+project doc/conversion ;
+
+import boostbook ;
+import quickbook ;
+
+xml conversion_lib : conversion.qbk ;
+boostbook conversion
+    :
+        conversion_lib
+    :
+        <xsl:param>boost.root=../../../..
+        # File name of HTML output:
+        <xsl:param>root.filename=conversion
+        # How far down we chunk nested sections, basically all of them:
+        <xsl:param>chunk.section.depth=0
+        # Don't put the first section on the same page as the TOC:
+        <xsl:param>chunk.first.sections=0
+        # How far down sections get TOC's
+        <xsl:param>toc.section.depth=2
+        # Max depth in each TOC:
+        <xsl:param>toc.max.depth=2
+        # How far down we go with TOC's
+        <xsl:param>generate.section.toc.level=0
+
+        <xsl:param>generate.manifest=0
+     ;
+
+###############################################################################
+alias boostdoc ;
```

Comment:
I'm not sure about those lines. Take a look at the https://github.com/boostorg/lexical_cast/blob/develop/doc/Jamfile.v2 and https://github.com/apolukhin/type_index/blob/develop/doc/Jamfile.v2

Looks like you must have the `boostbook standalone` line and different `alias boostdoc`.